config/playground/helm/dev/helm-opensearch-dashboards.yaml

```diff
@@ -0,0 +1,216 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for opensearch-dashboards.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+opensearchHosts: "https://opensearch-cluster-master:9200"
```

Comment:
let' replace replace non-inclusive terminology `master` to `leader`

Reply:
> let' replace replace non-inclusive terminology master to leader

I was able to change all the instances in `helm-opensearch-dashboards.yaml` but in `helm-opensearch.yaml`, I believe the `master` role is needed for OSD to function.

Reply:
Perhaps a future issue is needed to refactor the terminology to a more inclusive one.